src/main/scan/select.c

```diff
@@ -28,75 +28,75 @@
 #define TRACE()
 
 AerospikeScan *AerospikeScan_Select(AerospikeScan *self, PyObject *args,
-									PyObject *kwds)
+                                    PyObject *kwds)
 {
-	TRACE();
+    TRACE();
 
-	char *bin = NULL;
-	PyObject *py_ustr = NULL;
-	as_error err;
-	as_error_init(&err);
+    char *bin = NULL;
+    PyObject *py_ustr = NULL;
+    as_error err;
+    as_error_init(&err);
 
-	if (!self || !self->client->as) {
-		as_error_update(&err, AEROSPIKE_ERR_PARAM, "Invalid aerospike object");
-		goto CLEANUP;
-	}
+    if (!self || !self->client->as) {
+        as_error_update(&err, AEROSPIKE_ERR_PARAM, "Invalid aerospike object");
+        goto CLEANUP;
+    }
```

Comment:
the only changes in this file are whitespace
I think if whitespace were to be fixed, that should be a separate PR